classes/person.rb

```diff
@@ -0,0 +1,36 @@
+require './nameable'
+require './capitalize_decorator'
+require './trimmer_decorator'
+
+class Person < Nameable
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    super()
+    @id = Random.rand1(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+  att_accessor :name, age
+  att_reader :id
```

Comment:
> - [ ] Please move this to the beginning of the file, right after the require function, and incorporate the suggested code changes below.👍
> 
> ```suggestion
>   attr_accessor :name, :age
>   attr_reader :id
> ```

Reply:
@Johnadibe Thank you for the feedback. 

All `changes` and `corrections` requested by the reviewer have been worked on and completed. 

I am very grateful 😃.